File: sum_abs_diff/sum.js
// helper.js (template)
class Solution {
  /**
   * @param {bigint[]} arr - array of integers as BigInt
   * @return {bigint} sum of absolute differences across all unordered pairs
   */
  sumOfAbsDiff(arr) {
    // Write your code here
    let sum = 0;
    for(let i = 0; i<arr.length; i++) {
      let diff = 0;
      for(let j = i+1; j<arr.length; j++) {
        diff = arr[i] - arr[j];
        sum += Math.abs(Number(diff));
      }
    }
    return sum;
  }
}

module.exports = { Solution };

File: sum_abs_diff/sum.py
def sum_abs_diff(arr):
    sum = 0 
    # 0= < i < j
    for i in range(0, len(arr)-1): 
        for j in range(i+1, len(arr)):
            diff = arr[i] - arr[j]
            sum += abs(diff)
    return sum

print(sum_abs_diff([1, 2, 3, 4, 5]))
print(sum_abs_diff([-1, 3, -2, 5]))
